app/build.gradle

```diff
@@ -33,33 +33,38 @@ android {
     }
 
     buildTypes {
-        release {
+        debug {
+            shrinkResources false
+            minifyEnabled false
+            applicationIdSuffix ".debug"
+            manifestPlaceholders.isRaptorEnabled = "true"
+        }
+        release { // This isn't deployed to Google Play, but is used as a template for "real" releases like "beta"
             shrinkResources true
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            matchingFallbacks = ['release']
```

Comment:
Interesting that this works. This may need a comment though; otherwise it's confusing: Why does "release" need a fallback on itself?